data/json/hobbies.json

```diff
@@ -342,20 +503,51 @@
     "type": "profession",
     "subtype": "hobby",
     "id": "sewing",
-    "name": "Sewing",
+    "name": "Sewing (Intermediate)",
     "description": "You are quite good at sewing torn clothes back together, and have even made your own clothes a few times.",
     "points": 3,
     "proficiencies": [ "prof_closures", "prof_elastics", "prof_knitting" ],
     "skills": [ { "level": 3, "name": "tailor" }, { "level": 1, "name": "fabrication" } ]
   },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "sewing_expert",
+    "name": "Sewing (Expert)",
+    "description": "Despite modern technology making career tailors obsolete you've spent much of your life hand-making clothing of all sorts, whether for historical purposes or to sell as artisinal goods.  Too bad you can't stitch the holes in reality closed.",
+    "points": 8,
+    "proficiencies": [
+      "prof_closures",
+      "prof_elastics",
+      "prof_knitting",
+      "prof_knitting_speed",
+      "prof_quilting",
+      "prof_closures_waterproofing",
+      "prof_leatherworking_basic",
+      "prof_furriery",
+      "prof_spinning",
+      "prof_weaving"
+    ],
+    "skills": [ { "level": 7, "name": "tailor" }, { "level": 2, "name": "fabrication" } ]
+  },
   {
     "type": "profession",
     "subtype": "hobby",
     "id": "fencing",
-    "name": "Fencing",
+    "name": "Fencing (Intermediate)",
     "description": "You were in a fencing club, and you weren't too shabby with a saber.  If you can get your hands on a real sword, you'll be a formidable fighter.",
-    "points": 5,
-    "skills": [ { "level": 3, "name": "stabbing" }, { "level": 1, "name": "melee" } ],
+    "points": 6,
+    "skills": [ { "level": 3, "name": "stabbing" }, { "level": 3, "name": "melee" }, { "level": 3, "name": "dodge" } ],
+    "traits": [ "MARTIAL_FENCING" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "fencing_expert",
+    "name": "Fencing (Expert)",
+    "description": "You've spent so long practicing your fencing you wouldn't look out of place in the Renaissance dueling with ne'er-do-wells and rapscallions.  En Garde!",
+    "points": 10,
```

Comment:
10 points is a lot when compared to the 8 of diy_crafts_expert when you don't even get any proficency. Are these skills really good enough to justify 10 points? My suggestion is to lower it to 9 or raise the points of `diy_crafts_expert`. Alternatively, add a nice fencing sword or other items to ```fencing_expert```

Reply:
Expert fencing and expert melee training have higher point values simply because starting with higher levels in combat skills is VASTLY more useful than crafting skills. Not that points matter anymore beyond just sorting the menu, but I digress. A character that starts with 5 in their melee skills (which previously was firmly mid to early lategame level of skill) can absolutely trounce early game enemies like zombies, even with no armor and early weapons. Add on how much of a power magnifier melee martial arts are and that's why the point costs are high.